package.json

```diff
@@ -11,6 +11,7 @@
     "react-dom": "^17.0.2",
```

Comment:
- [ ] It would be better to import `Provider` in the `src/index.js`  file as recommended since this is the entry point of the application. It would also be better if we wrap the App component with the `Provider` as you did in the componentsApp.js file